### ISSUE TITLE
Manifest property for the license of the packages themselves? (not the packaged program)

### ISSUE THREAD

**Issue by fpesari** [flatpak/flatpak#4247]
Hello,

Is there a property for the license of the package itself? I am not talking about the packaged programs, I am talking about the manifest and all files written by the Flathub packagers.

In absence of such information, those files are to be considered proprietary under a full copyright and as such, can't be modified, redistributed, studied or used without the explicit consent of the author.

In other words, this means that a repository (like Flathub) can go down one day and nobody would be legally allowed to restore it :crying_cat_face:

For this reason, I think that if such a field does not exist, it should exist and be made obligatory for a valid manifest.

**Comment by fpesari** [comment]
Update: could this property be [metadata_license](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-metadata_license)? If that's the case, I never encountered inside an actual package. If so, I think both `metadata_license` and `project_license` should be made obligatory, as having no license [means proprietary by default](https://choosealicense.com/no-permission/) and that would harm community projects like Flathub in the long-run.

**Comment by hfiguiere** [comment]
You can add a comment in the manifest to specify the license. You can use SPDX do be machine readable.
